packages/components/src/toolbar/test/index.js

```diff
@@ -1,110 +0,0 @@
-/**
- * External dependencies
- */
-import { shallow } from 'enzyme';
-
-/**
- * Internal dependencies
- */
-import Toolbar from '../';
-
-describe( 'Toolbar', () => {
```

Comment:
Maybe we should keep them for backward compatibility with the note in the `describe` that it's a deprecated version?

In each test we would have to add:

```js
expect( console ).toHaveWarnedWith( 'Using `Toolbar` as a collapsible group of controls...' );
```

Reply:
It’s probably fine to keep one of those tests only. I figured out you moved the existing tests to the suite which covers the toolbar group. It didn’t occur to me at first.